### PR TITLE
DOC: formatting updates

### PR DESCRIPTION
**Pull request by kecnry**
### Description

This pull request fixes broken/missing links, outdated UI layout references, minor typos and formatting issues, and adds inline icon images to the docs.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a change log needed? If yes, is it added to `CHANGES.rst`?
- [ ] Is a milestone set? Milestone is only currently required for PRs related to Imviz MVP.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

**Comment by larrybradley** [comment]
RTD build is failing due to an undefined label in `displaycubes.rst`:
`/home/docs/checkouts/readthedocs.org/user_builds/jdaviz/checkouts/887/docs/cubeviz/displaycubes.rst:200: WARNING: undefined label: #plot-settings`

File: docs/cubeviz/index.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@
 Cubeviz
 #######
 
-Cubeviz is a visualization and analysis toolbox for data cubes from integral field units (IFUs). It is built as part of the `Glue visualiztion <https://glueviz.org>`_ tool. Cubeviz is designed to work with data cubes from the NIRSpec and MIRI instruments on JWST, and will work with IFU data cubes. It uses the `specutils <https://specutils.readthedocs.io/en/latest/>`_ package from `Astropy <https://www.astropy.org>`_ .
+Cubeviz is a visualization and analysis toolbox for data cubes from integral field units (IFUs). It is built as part of the `Glue visualization <https://glueviz.org>`_ tool. Cubeviz is designed to work with data cubes from the NIRSpec and MIRI instruments on JWST, and will work with IFU data cubes. It uses the `specutils <https://specutils.readthedocs.io/en/latest/>`_ package from `Astropy <https://www.astropy.org>`_ .
```
Comment (pllim) [inline docs/cubeviz/index.rst:11]: @larrybradley , how did you not catch this? 😆

File: docs/cubeviz/plugins.rst [hunk 2]
```diff
@@ -147,7 +148,7 @@ integrates the flux density along the spectral axis to compute
 a moment map. The order of the moment map (0, 1, 2, ...) indicates
 the power-law index to which the spectral axis is raised.  A 'moment 0' map
 gives the integrated flux over a spectral region. Similarly, 'moment 1'
-is the flux-weighted centroid (e.g. line center) and a 'moment 2'
+is the flux-weighted centroid (e.g. line center) and 'moment 2'
```
Comment (pllim) [inline docs/cubeviz/plugins.rst:151]: Might as well?
```suggestion
is the flux-weighted centroid (e.g., line center) and 'moment 2'
```

**Review by pllim — APPROVED** [review]
One tiny nitpick and is kinda unrelated, so I am approving. I think we should get this in for the release. Nice cleanup! 👏

**Comment by pllim** [comment]
Sorry for the conflict. Please rebase.

**Review by rosteen — APPROVED** [review]
Looks great, love the inline icons in the text descriptions. Thanks!

File: docs/cubeviz/index.rst (hunk 1 above)
Comment (larrybradley) [reply docs/cubeviz/index.rst:11]: Ha, I wasn't actually *reading* the docs. 😆  It appears `codespell` doesn't catch everything.

**Pull request merged**